lib/proofer/result.rb

```diff
@@ -47,6 +47,10 @@ def success?
       !exception? && !errors?
     end
 
+    def timed_out?
+      exception? && @exception.is_a?(Proofer::TimeoutError)
```

Comment:
Style: `@exception&.is_a?(Proofer::TimeoutError)`?

Reply:
I actually don't think we need to guard against nil here b/c `Nilclass` is not an instance of `Proofer::TimeouError`. I'll tie that up before merging.